docs/topics/getting-started.md

```diff
@@ -166,7 +166,52 @@ Follow Kotlin on ![Twitter](twitter.svg){width=25}{type="joined"} [Twitter](http
 
 </tab>
 
+<tab title="Multiplatform library">
+
+Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the same code for different platforms while retaining the flexibility and benefits of native programming.
+
+Here you'll learn how to develop and publish a multiplatform library:
+
+1. **Install the [latest version of IntelliJ IDEA](http://www.jetbrains.com/idea/download/index.html).**
+
+2. **Create a multiplaform library:**
+
+   * To start from scratch, [create a basic project](mpp-create-lib.md).
+   * If you prefer more robust examples, complete the [Create and publish a multiplatform library](multiplatform-library.md) tutorial. It shows how to create a multiplatform library for JVM, JS, and Native platforms, test it and publish to a local Maven repository.
+   * Build a full stack web application using [this hands-on](https://play.kotlinlang.org/hands-on/Full%20Stack%20Web%20App%20with%20Kotlin%20Multiplatform/01_Introduction) hands-on.
+
+3. **Use libraries in your application.** Learn more about [adding dependencies on libraries](mpp-add-dependencies.md).
+
+   |Library|Details|
+   |-------|-------|
+   | Ktor |  [Docs](https://ktor.io/learn/) and [sample](https://play.kotlinlang.org/hands-on/Full%20Stack%20Web%20App%20with%20Kotlin%20Multiplatform/03_A_Simple_API_Server).| 
```

Comment:
Maybe it's worth editing the table in Cross-platform mobile app section too? Because there is the same problem with the links.

Reply:
Sure, it's a good idea for the next PR :)